orchagent/mirrororch.cpp

```diff
@@ -613,6 +731,92 @@ bool MirrorOrch::updateSession(const string& name, MirrorEntry& session)
     return ret;
 }
 
+bool MirrorOrch::setUnsetPortMirror(Port port,
+                                    bool ingress,
+                                    bool set,
+                                    sai_object_id_t sessionId)
+{
+    sai_status_t status;
+    sai_attribute_t port_attr;
+    port_attr.id = ingress ? SAI_PORT_ATTR_INGRESS_MIRROR_SESSION:
+                           SAI_PORT_ATTR_EGRESS_MIRROR_SESSION;
+    if (set)
+    {
+        port_attr.value.objlist.count = 1;
+        port_attr.value.objlist.list = (long unsigned int *)calloc(port_attr.value.objlist.count, sizeof(sai_object_id_t));
+        port_attr.value.objlist.list[0] = sessionId;
+    }
+    else
+    {
+        port_attr.value.objlist.count = 0;
+    }
+
+    if (port.m_type == Port::LAG)
+    {
+        vector<Port> portv;
+        m_portsOrch->getLagMember(port, portv);
+        for (const auto p : portv)
+        {
+            status = sai_port_api->set_port_attribute(p.m_port_id, &port_attr);
+            if (status != SAI_STATUS_SUCCESS)
+            {
+                SWSS_LOG_ERROR("Failed to configure %s session on port %s: %s, status %d, sessionId %x\n",
+                                ingress ? "RX" : "TX", port.m_alias.c_str(),
+                                p.m_alias.c_str(), status, sessionId);
+                return false;
+            }
+        }
+    }
+    else if (port.m_type == Port::PHY)
+    {
+        status = sai_port_api->set_port_attribute(port.m_port_id, &port_attr);
+        if (status != SAI_STATUS_SUCCESS)
+        {
+            SWSS_LOG_ERROR("Failed to configure %s session on port %s, status %d, sessionId %x\n",
```

Comment:
You've used `\n` inconsistently. Some places have and some not.` \n` is not required, kindly remove

Reply:
Yes. taken care of this. There were some existing code also wrongly doing .fixed those too.